### ISSUE TITLE
Help you to fix recent crash of rpi.gpio install

### ISSUE THREAD

**Issue by Elrondo46** [kounch/argonone#7]
Replace this by that to fix the bug:
```bash 
/opt/argonone/bin/python3 -m pip install pysmbus RPi.GPIO
```
```bash
env CFLAGS="-fcommon" /opt/argonone/bin/python3 -m pip install pysmbus RPi.GPIO
```

in install file

Regards

**Comment by dkimmortal** [comment]
I tried this but it didn’t fix it, would it work ok with aarch64 install instead of armv7 Arch? 
Do I have to manually uninstall all the packages before doing an install with this fix? I’m not sure what I’m doing wrong

**Comment by SpacingBat3** [comment]
@Elrondo46, could you just do PR (Pull Request) and describe a little more what it fixes? Is it a fix for issue #8 ?
